packages/client/tests/functional/referential-action-set-null/tests_1-to-1.ts

```diff
@@ -0,0 +1,405 @@
+import { setupMSSQL } from '../../../src/utils/setupMSSQL'
+import type { ProviderFlavor } from './_matrix'
+import testMatrix from './_matrix'
+import * as mssql from './_utils/mssql'
+import * as mysql from './_utils/mysql'
+import * as postgres from './_utils/postgres'
+import type { SetupParams } from './_utils/types'
+
+// @ts-ignore
+const describeIf = (condition: boolean) => (condition ? describe : describe.skip)
+
+// @ts-ignore
+const testIf = (condition: boolean) => (condition ? test : test.skip)
+
+function getDatabaseURL(providerFlavor: ProviderFlavor) {
+  switch (providerFlavor) {
+    case 'mssql':
+      return 'mssql://SA:Pr1sm4_Pr1sm4@localhost:1433/PRISMA_DB_NAME'
```

Comment:
We could use from the GitHub Actions `TEST_MSSQL_URI: mssql://SA:Pr1sm4_Pr1sm4@localhost:1433/master`